src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -22,6 +23,70 @@
 )
 
 
+class ScanImageTiffMultiPlaneMultiFileImagingExtractor(MultiImagingExtractor):
+    """Specialized extractor for reading multi-file (buffered) TIFF files produced via ScanImage."""
+
+    extractor_name = "ScanImageTiffMultiPlaneMultiFileImaging"
+    is_writable = True
+    mode = "folder"
+
+    def __init__(self, folder_path: PathType, file_pattern: str, channel_name: str) -> None:
+        """Create a ScanImageTiffMultiPlaneMultiFileImagingExtractor instance from a folder of TIFF files produced by ScanImage.
+
+        Parameters
+        ----------
+        folder_path : PathType
+            Path to the folder containing the TIFF files.
+        file_pattern : str
+            Pattern for the TIFF files to read -- see pathlib.Path.glob for details.
+        channel_name : str
+            Channel name for this extractor.
+        """
+        self.folder_path = Path(folder_path)
+        file_paths = sorted(self.folder_path.glob(file_pattern))
```

Comment:
I think we should use `natsort` here and in `ScanImageTiffSinglePlaneMultiFileImagingExtractor` as well if you agree,
```suggestion
        from natsort import natsorted
        
        self.folder_path = Path(folder_path)
        file_paths = natsorted(self.folder_path.glob(file_pattern))
```